src/main/kotlin/org/wycliffeassociates/otter/common/domain/md/ParseMd.kt

```diff
@@ -0,0 +1,56 @@
+package org.wycliffeassociates.otter.common.domain.md
+
+import java.io.BufferedReader
+
+// TODO: Add Help type enum to HelpResource? (tn, tq)
+data class HelpResource(var title: String, var body: String)
+
+class HelpResourceList: ArrayList<HelpResource>()
```

Comment:
[Type aliases](https://kotlinlang.org/docs/reference/type-aliases.html) might be better than subclassing here.  I'm new to Kotlin, of course, so I don't know.  It may be best just not to bother renaming to `HelpResourceList`, and just use `List<HelpResource>`.

Reply:
Please use List<HelpResource> (or MutableList<HelpResource> if that's needed), it's consistent with how we use lists everywhere else in the codebase.

I hadn't seen type aliasing before, so thanks for that!

Reply:
Type aliases looks like the way to go here compared to subclassing. But now I think it might be better not to even bother creating HelpResourceList, since List<HelpResource> is actually more straightforward and not very complicated.